.github/workflows/be_discovery_service.yml

```diff
@@ -30,8 +30,23 @@ jobs:
          architecture: x64
          
       - name: Build with maven
-        run: mvn dependency:tree clean install
+        run: mvn clean install
```

Comment:
Why did we removed this flag from the command? 
The log should be verbose for issues to be spotted